### PR TITLE
Yoav/singleton

### PR DESCRIPTION
**Pull request by yoav-el-certora**

### CODE REVIEW

**Comment by yoav-el-certora** [comment]
https://prover.certora.com/output/99668/f20ede8406054d28b04b5fc89b3e90da?anonymousKey=8eeae5106f78fbfff854141cf889cb7cf6f08857

This rule should be executed without all function added in webAuthnHarness so and it wont get timeout

File: certora/specs/SafeWebAuthnSignerSingleton.spec [hunk 1]
```diff
@@ -46,6 +54,7 @@ rule verifySignatureUniqueness(env e){
     bytes4 second_message_verified = isValidSignature(e, second_message, signature);
 
     assert (first_message != second_message) => !(first_message_verified == MAGIC_VALUE() && second_message_verified == MAGIC_VALUE());
+    satisfy true;
```
Comment (nivcertora) [inline certora/specs/SafeWebAuthnSignerSingleton.spec:57]: Why do we need those satisfy true here?
It's not the same as using rule_sanity basic?

**Review by nivcertora — CHANGES_REQUESTED** [review]
Small comment

File: certora/specs/SafeWebAuthnSignerSingleton.spec (hunk 1 above)
Comment (yoav-el-certora) [reply certora/specs/SafeWebAuthnSignerSingleton.spec:57]: It is, but Ive added it to have also a call trace for a valid run.

**Review by nivcertora — APPROVED** [review]

**Pull request merged**